src/app/inventory/selectors.ts

```diff
@@ -106,25 +109,83 @@ export const materialsSelector = (state: RootState) =>
 /** The actual raw profile response from the Bungie.net profile API */
 export const profileResponseSelector = (state: RootState) => state.inventory.profileResponse;
 
+const STORE_SPECIFIC_OWNERSHIP_BUCKETS = [
+  // Emblems cannot be transferred between characters and if one character owns an emblem,
+  // other characters don't really own it. Also affects vendor claimability.
+  BucketHashes.Emblems,
+  // Quests and bounties are character-specific.
+  BucketHashes.Quests,
+];
+
+/**
+ * For most items, ownership is simple -- if it's in any bucket, we own this item.
+ * For items in some buckets (like quests and emblems), there is no concept of
+ * account-wide ownership. This mostly affects vendor claimability.
+ */
+export interface OwnedItems {
```

Comment:
We were less clear about what "ownership" meant at this point - it's worth clarifying that this is ownership as in "Has the item in inventory" vs maybe "unlocked" meaning that you have it in collections.